From f329b4a899562497610b4da42ea4c26b93e4d52f Mon Sep 17 00:00:00 2001
From: Corinna Vinschen <corinna@vinschen.de>
Date: Tue, 1 Mar 2022 14:06:09 +0100
Subject: [PATCH 034/N] Cygwin: /proc/<PID>/status: fix format specifier

The format specifier used for the signal masks is incorrectly
assuming 32 bit values.  Fix it.

Signed-off-by: Corinna Vinschen <corinna@vinschen.de>
Signed-off-by: Johannes Schindelin <johannes.schindelin@gmx.de>
---
 winsup/cygwin/fhandler_process.cc | 6 +++---
 1 file changed, 3 insertions(+), 3 deletions(-)

diff --git a/winsup/cygwin/fhandler_process.cc b/winsup/cygwin/fhandler_process.cc
index 888604e..b5bc0e4 100644
--- a/winsup/cygwin/fhandler_process.cc
+++ b/winsup/cygwin/fhandler_process.cc
@@ -1257,9 +1257,9 @@ format_process_status (void *data, char *&destbuf)
 				   "VmStk:\t%8lu kB\n"
 				   "VmExe:\t%8lu kB\n"
 				   "VmLib:\t%8lu kB\n"
-				   "SigPnd:\t%016x\n"
-				   "SigBlk:\t%016x\n"
-				   "SigIgn:\t%016x\n",
+				   "SigPnd:\t%016lx\n"
+				   "SigBlk:\t%016lx\n"
+				   "SigIgn:\t%016lx\n",
 			  cmd,
 			  state, state_str,
 			  p->pgid,
-- 
2.9.0

